sinks/ssfmetrics/metrics.go

```diff
@@ -23,6 +23,8 @@ type metricExtractionSink struct {
 	spansProcessed         int64
 	metricsGenerated       int64
 	parser                 samplers.Parser
+	// TODO add some "additionalTags" field here so that we can identify
```

Comment:
We have diminished introspection ability here and it would be nice to improve this.

Reply:
Resolving thread, we made `ComputationRoutingConfig` a first class struct field so that we could apply `MatcherConfigs` within ssfmetrics.

Reply:
(Although ssfmetrics is a sink, it's also an entrypoint into workers themselves)